themes/dco/templates/individual--foaf-person.ftl

```diff
@@ -112,7 +115,7 @@
         <#include "individual-researchAreas.ftl">
 
         <!-- Geographic Focus -->
-        <#include "individual-geographicFocus.ftl">
+        <!-- #include "individual-geographicFocus.ftl" -->
```

Comment:
I agree. Let's go ahead and just remove the comment and the commented out include.